### ISSUE TITLE
efp.matrix: implement lrvar and vcov parameters

### ISSUE THREAD

**Issue by GreatEmerald** [appelmar/strucchange#32]
@appelmar Could you check this part? Achim implemented two optional parameters to efp[.formula], could this be ported to the matrix function as well? Or maybe we should merge the two functions so that changes like that are easier to integrate?

_Originally posted by @GreatEmerald in https://github.com/bfast2/strucchangeRcpp/pull/29#discussion_r584194275_

**Comment by appelmar** [comment]
AFAICS, integrating `lrvar` would be straightforward for types `"Rec-CUSUM", "OLS-CUSUM", "Rec-MOSUM", and "OLS-MOSUM"` but more difficult for `RE` and `ME`.  Integrating `vcov` seems even more complex as it is a function which would need to be called from the C++ code (since it is only used for types `RE` and `ME`). Shall I try to add `lrvar`? Maybe we can still add both arguments but throw a warning if it is not implemented?

**Comment by GreatEmerald** [comment]
Yes, I think the way to go is that if it is too complex, we can follow upstream, which is to say ignore it with a warning. Handling it in some way is useful, as otherwise users might be confused as to why it's not working for them when using matrices, but works when using the formula method.
